internal/interface/interface.go

```diff
@@ -0,0 +1,45 @@
+package iface
```

Comment:
nit: Let's also just name the directory/file to `iface` as well.  The boilerplate should also probably go above any declarations.